UNL-CSE-AMBASSADORS/VITA
# Set up document upload for one Azure directory per site

To test this locally:

- Log in to portal.azure.com (you may need to accept an email invite first, subject line "UNL VITA (Tax Credit Campaign) invited you to access applications within their organization"). After logging in, make sure you see "UNL VITA (Tax Credit Campaign)" under your name on the right side.
- Select the unlvitadocuments storage account and find Configuration on the left side. Disable "secure transfer required" to allow for transfer from your localhost on HTTP.
- Also on the left side of the unlvitadocuments Storage Account page, find Access Keys, get the connection string and paste it in config.php. In the string, change https to http.
- I think you may need to make PROD = true in config.php.
- **Make sure to sign up under a name that is a conspicuous test alias like "TestFirst TestLast."**

To test this, I hard-coded siteIds from 1-14 (only 13 exist) to see if it could connect to every site's folder. You can see in ISS's folder that it's able to upload more than one document to a folder.

Container names are currently hard-coded based on siteId. We could refactor this code to include lookupNames for Site table, as has been done for Permission and AppointmentType.
